subsys/bluetooth/host/iso.c

```diff
@@ -1859,16 +1863,28 @@ static bool valid_cig_param(const struct bt_iso_cig_param *param, bool advanced,
 		return false;
 	}
 
-	if (param->interval < BT_ISO_SDU_INTERVAL_MIN ||
-	    param->interval > BT_ISO_SDU_INTERVAL_MAX) {
-		LOG_DBG("Invalid interval: %u", param->interval);
+	if (param->c_to_p_interval < BT_ISO_SDU_INTERVAL_MIN ||
+	    param->c_to_p_interval > BT_ISO_SDU_INTERVAL_MAX) {
```

Comment:
Could also modify these to use `IN_RANGE` while we are modifying them now

Reply:
done